src/dnx.coreclr.unix/dnx.coreclr.cpp

```diff
@@ -233,6 +237,113 @@ int InvokeDelegate(host_main_fn host_main, int argc, const char** argv)
     return exit_code;
 }
 
+#if defined(PLATFORM_LINUX)
+
+std::string get_os_version()
+{
+    std::vector<std::string> qualifiers { "DISTRIB_ID=", "DISTRIB_RELEASE=" };
+
+    std::ifstream lsb_release;
+    lsb_release.open("/etc/lsb-release", std::ifstream::in);
+    if (lsb_release.is_open())
```

Comment:
What do you mean by "OS Name and Version"? OS Name has a separate property but the value returned by this function goes to OsVersion. So in this case you will get:

```
Information: Operating System: Linux
Information: Operating System Version: Ubuntu 15.04
```